pig/src/main/resources/rh-license-exceptions.json

```diff
@@ -794,7 +794,7 @@
   {
     "groupId": "io.agroal",
     "artifactId": "agroal-api",
-    "version-regexp": "1.8*",
+    "version-regexp": "1\\.8*",
```

Comment:
this looks a bit suspicious. Shouldn't it be sth like `1\\.8.*` ? (just a note, this change doesn't actually change the suspicious part)

Reply:
I'll have a look. Thanks for pointing this out!

Reply:
agroal actually has version `1.8`, `1.9` etc.

`1\\.8.*` makes sense in that optic. Will do the change!

Reply:
I would probably go for `1\\.8(\\..*)?` to not include e.g. 1.80 in a few years. WDYT?